tests/subworkflows/icgc-argo-workflows/merge_dup/test.yml

```diff
@@ -0,0 +1,49 @@
+- name: "merge_dup_markdup"
+  command: nextflow run ./tests/subworkflows/icgc-argo-workflows/merge_dup -entry test_merge_dup --tools 'markdup' -profile docker -c ./tests/config/nextflow.config
+    tags:
+    - "subworkflows"
+    - "subworkflows/merge_dup"
+  files:
+    - path: "output/samtools_merge/Study123.PatientXYZ.SampleA.bam"
+    - path: "output/biobambam_bammarkduplicates2/Study123.PatientXYZ.SampleA.csort.markdup.bam"
+    - path: "output/biobambam_bammarkduplicates2/Study123.PatientXYZ.SampleA.csort.markdup.metrics.txt"
+    - path: "output/samtools_index/Study123.PatientXYZ.SampleA.csort.markdup.bam.bai"
+    - path: "output/samtools_convert/Study123.PatientXYZ.SampleA.csort.markdup.cram"
+    - path: "output/samtools_convert/Study123.PatientXYZ.SampleA.csort.markdup.cram.crai"
+    - path: "output/tar/out/Study123.PatientXYZ.SampleA.RNA-seq.aln.cram.duplicates_metrics.tgz"
+
+- name: "merge_dup_default"
+  command: nextflow run ./tests/subworkflows/icgc-argo-workflows/merge_dup -entry test_merge_dup -profile docker -c ./tests/config/nextflow.config
+    tags:
+    - "subworkflows"
+    - "subworkflows/merge_dup"
+  files:
+    - path: "output/samtools_merge/Study123.PatientXYZ.SampleA.bam"
+    - path: "output/samtools_index/Study123.PatientXYZ.SampleA.bam.bai"
+    - path: "output/samtools_convert/Study123.PatientXYZ.SampleA.csort.cram"
+    - path: "output/samtools_convert/Study123.PatientXYZ.SampleA.csort.cram.crai"
+
+- name: "merge_dup_markdup_transcript"
+  command: nextflow run ./tests/subworkflows/icgc-argo-workflows/merge_dup -entry test_merge_dup_transcript --tools 'markdup' -profile docker -c ./tests/config/nextflow.config
```

Comment:
What is the purpose of this test? It outputs the same test file regardless. If looking to demonstrate transcriptome renaming, adding in the arg to a different config file.